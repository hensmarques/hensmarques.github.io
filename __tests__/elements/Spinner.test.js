import React from 'react';
import Spinner from '../../src/elements/Spinner';

describe('<Spinner/>', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<Spinner />);
    expect(wrapper).toMatchSnapshot();
  });
});
