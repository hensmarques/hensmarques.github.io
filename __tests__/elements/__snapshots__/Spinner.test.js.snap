// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner/> renders and matches the snapshot 1`] = `
<div
  className="loader-animation "
>
  <div
    className="lds-ring"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
